src/grpc.ts

```diff
@@ -24,7 +24,7 @@ import {join} from 'path';
 import {OutgoingHttpHeaders} from 'http';
 import * as path from 'path';
 import * as protobuf from 'protobufjs';
-import * as objectHash from 'object-hash';
+import * as crypto from 'crypto';
```

Comment:
This is harder than that. We cannot just import `crypto` because it's a Node.js specific module. To be compatible with browsers, we need to be able to use `crypto` for Node and `window.crypto.subtle` for browsers.

See example here: https://github.com/googleapis/google-auth-library-nodejs/tree/main/src/crypto

Reply:
Synced offline. Add issue #1199 for long term consideration.